src/components/partner/partner.resolver.ts

```diff
@@ -93,6 +95,28 @@ export class PartnerResolver {
     );
   }
 
+  @ResolveField(() => SecuredLocations)
+  async countries(
+    @Parent() partner: Partner,
+    @Loader(LocationLoader) loader: LoaderOf<LocationLoader>,
+  ): Promise<SecuredLocations> {
+    const countries = (await loader.loadMany(partner.countries.value)).flatMap(
+      (country) => {
+        if (country instanceof NotFoundException) {
+          return [];
+        } else if (country instanceof Error) {
+          return country;
```

Comment:
In this case `country` is actually an `Error`, so you want to throw it instead of return it:

```typescript
throw country;
```

This will take care of your build error.